conan/tools/gnu/autotools.py

```diff
@@ -48,16 +49,19 @@ def make(self, target=None, args=None):
         str_args = self._make_args
         str_extra_args = " ".join(args) if args is not None else ""
         jobs = ""
-        if "-j" not in str_args and "nmake" not in make_program.lower():
+        jobs_already_passed = re.search(r"-j(\d+)", join_arguments([str_args, str_extra_args]))
```

Comment:
it's still a little bit fragile, a pattern like `/users/foo-j47/...` may be detected.

a regex like this one `(^-j\d+$)|(^-j\d+\s+)|(\s+-j\d+$)|(\s+-j\d+\s+)` for example?